CMake/resolve_dependency_modules/cudf.cmake

```diff
@@ -17,35 +17,55 @@ include_guard(GLOBAL)
 # 3.30.4 is the minimum version required by cudf
 cmake_minimum_required(VERSION 3.30.4)
 
-set(VELOX_rapids_cmake_VERSION 25.04)
+set(VELOX_rapids_cmake_VERSION 25.08)
 set(VELOX_rapids_cmake_BUILD_SHA256_CHECKSUM
-    458c14eaff9000067b32d65c8c914f4521090ede7690e16eb57035ce731386db)
+    f0e5484d49d9365b84eb41799db4fb1c5c8532f251ace734ead168bbbc1111c7)
 set(VELOX_rapids_cmake_SOURCE_URL
-    "https://github.com/rapidsai/rapids-cmake/archive/7828fc8ff2e9f4fa86099f3c844505c2f47ac672.tar.gz"
+    "https://github.com/rapidsai/rapids-cmake/archive/492d474a62a91dd61bd6b91994eec2125e12fab8.tar.gz"
 )
 velox_resolve_dependency_url(rapids_cmake)
 
-set(VELOX_rmm_VERSION 25.04)
+set(VELOX_rmm_VERSION 25.08)
 set(VELOX_rmm_BUILD_SHA256_CHECKSUM
-    294905094213a2d1fd8e024500359ff871bc52f913a3fbaca3514727c49f62de)
+    9dc014c44191b0a23430f1bd7ebe38663643cdbbedd47a5a8e2079fdbbd4eb2e)
 set(VELOX_rmm_SOURCE_URL
-    "https://github.com/rapidsai/rmm/archive/d8b7dacdeda302d2e37313c02d14ef5e1d1e98ea.tar.gz"
+    "https://github.com/rapidsai/rmm/archive/a68fdfe26e0d1e2e37feddf53eda1b7b5044f9f4.tar.gz"
 )
 velox_resolve_dependency_url(rmm)
 
-set(VELOX_kvikio_VERSION 25.04)
+set(VELOX_kvikio_VERSION 25.08)
 set(VELOX_kvikio_BUILD_SHA256_CHECKSUM
-    4a0b15295d0a397433930bf9a309e4ad2361b25dc7a7b3e6a35d0c9419d0cb62)
+    8a5251ee1dff576578b0f96130ef87bfd8693de4769c08d33f203b6ea6750bab)
 set(VELOX_kvikio_SOURCE_URL
-    "https://github.com/rapidsai/kvikio/archive/5c710f37236bda76e447e929e17b1efbc6c632c3.tar.gz"
+    "https://github.com/rapidsai/kvikio/archive/54c420a652bf29e3c7de9ac2e2e19af07de7c256.tar.gz"
 )
 velox_resolve_dependency_url(kvikio)
 
-set(VELOX_cudf_VERSION 25.04)
+set(VELOX_cudf_VERSION
+    25.08
+    CACHE STRING "cudf version")
+
+# cudf 25.08+ requires C++20
+if(VELOX_cudf_VERSION VERSION_GREATER_EQUAL "25.08")
```

Comment:
I don’t expect us to support versions earlier than those pinned here. I think it might be safe to eliminate this conditional entirely and always use C++20. See also the comments below.